src/layouts/MainLayout.js

```diff
@@ -47,7 +47,9 @@ const MainLayout = ({ data = {}, children, pageContext }) => {
 
   return (
     <>
-      <SEO location={location} />
+      <SEO location={location}>
+        <meta name="google-site-verification" content="foobar" />
```

Comment:
It doesn't look like the `SEO` component accepts a `children` prop. I also noticed the `foobar` set as `content`. I assume this was likely for testing purposes. Could you remove this?

Reply:
Sorry...I had that for testing and forgot to remove it. This should be resolved now.